models/reviews.models.js

```diff
@@ -47,9 +47,32 @@ exports.selectComments = (review_id) => {
 		return db
 			.query(
 				`SELECT * FROM comments
-        WHERE review_id = $1`,
+        WHERE review_id = $1;`,
 				[review_id]
 			)
 			.then((comments) => comments.rows);
 	});
 };
+
+exports.insertReviewComment = (review_id, newComment) => {
+	const { author, body } = newComment;
+	const newDate = new Date();
+	if (review_id === undefined) {
+		return Promise.reject({
+			status: 400,
+			msg: `Sorry, that's a bad request`,
+		});
+	}
+	return checkReviewExists(review_id).then(() => {
+		return db
+			.query(
+				`INSERT INTO comments
+        (author, body, review_id, votes, created_at)
+        VALUES
+        ($1, $2, $3, 0, $4)
+        RETURNING *;`,
+				[author, body, review_id, newDate]
+			)
+			.then((result) => result.rows[0]);
+	});
+};
```

Comment:
We don't need to insert a newDate or votes as 0 as these are being done by default in our seed file. Fully understand why you added these though as you didn't write the seed file and may not have known about the default values!